lib/core/viewer/components/pdfViewer.component.ts

```diff
@@ -150,7 +149,9 @@ export class PdfViewerComponent implements OnChanges, OnDestroy {
     }
 
     ngOnDestroy() {
-        window.document.removeEventListener('scroll', this.onDocumentScroll, true);
+        if(this.documentContainer) {
+            this.documentContainer.removeEventListener('pagechange', (event) => this.onPageChange(event), true);
```

Comment:
That should build a new anonymous function, not reference the previously set one ;)